cmd/internal/planetscale_edge_database.go

```diff
@@ -189,9 +195,12 @@ func (p PlanetScaleEdgeDatabase) Read(ctx context.Context, w io.Writer, ps Plane
 		if lcErr != nil {
 			return currentSerializedCursor, errors.Wrap(err, "Unable to get latest cursor position")
 		}
+		if latestCursorPosition == "" {
+			return currentSerializedCursor, errors.Wrap(err, "Unable to get latest cursor position")
+		}
 
 		// the last synced VGTID is not at least, or after the current VGTID
-		if currentPosition.Position != "" && !positionAtLeast(latestCursorPosition, currentPosition.Position) {
+		if currentPosition.Position != "" && !positionAfter(latestCursorPosition, currentPosition.Position) {
```

Comment:
if the start position is not after the stop position, exit the sync.

Reply:
is `!positionAfter` compatible with the two positions being identical? reads that way.

Reply:
Yeah - if they are equal, `!positionAfter` will return true (`positionAfter` will be false)